spec/acceptance/mysql_spec.rb

```diff
@@ -1,6 +1,6 @@
 require 'spec_helper_acceptance'
 
-describe 'jira mysql', unless: UNSUPPORTED_PLATFORMS.include?(fact('osfamily')) do
+describe 'jira mysql' do
   it 'installs with mysql database' do
     pp = <<-EOS
       if versioncmp($facts['puppetversion'],'3.6.1') >= 0 {
```

Comment:
I didn't want to clean this up, but there's obviously room for more improvements here.